src/codegen/sys/statics.rs

```diff
@@ -45,10 +45,6 @@ pub fn only_for_glib(w: &mut Write) -> Result<()> {
         "pub type gconstpointer = *const c_void;",
         "pub type gpointer = *mut c_void;",
         "",
-        "#[repr(C)]",
-        "#[derive(Copy, Clone)]",
-        "pub struct Volatile<T>(T);",
```

Comment:
This is not equivalent to `T` at the FFI level?

Reply:
Usually, but not always. For some examples see discussion about bitflags in #575.
It would be equivalent with `#[repr(transparent)]`, which is another nice but unstable thing.

Reply:
Ok, so this is only about wrappers around "fundamental" types and not structs